### PR TITLE
[luxtronikheatpump] Update channel descriptions in README.md

### PR DESCRIPTION
**Pull request by martin1**
temperatureHotWaterTarget channel is writable
temperatureTargetHotWater_2  channel doesn't exist in the binding

Signed-off-by: Martin Kurgi <martinkurgi@gmail.com>

### CODE REVIEW

File: bundles/org.openhab.binding.luxtronikheatpump/README.md [hunk 1]
```diff
@@ -253,7 +252,7 @@ The following channels are also writable:
 | channel  | type   | advanced | description                  |
 |----------|--------|----------|------------------------------|
 | temperatureHeatingParallelShift | Number:Temperature |   | Heating temperature (parallel shift) |
-| temperatureTargetHotWater_2 | Number:Temperature |   | Hot water temperature |
+| temperatureHotWaterTarget | Number:Temperature |   | Hot water target temperature |
```
Comment (wborn) [inline bundles/org.openhab.binding.luxtronikheatpump/README.md:255]: According to the [channel type](https://github.com/openhab/openhab-addons/blob/cfea5a2185929a5a2172d7c5159232b5a2b5c5ae/bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml#L63-L68) it is read only.

Comment (martin1) [reply bundles/org.openhab.binding.luxtronikheatpump/README.md:255]: I did some further investigation and found that the `temperatureHotWaterTarget` channel is actually defined as a writable channel [(Link)](https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.luxtronikheatpump/src/main/java/org/openhab/binding/luxtronikheatpump/internal/enums/HeatpumpChannel.java#L1196-L1211). So the channel type seems to be incorrect.
It is interesting however that the readonly channel type of `temperatureHotWaterTarget` does not stop me from writing to it.

Tested using heatpump type `MSW 12` on openHAB `3.1.0.M4`.

Comment (wborn) [reply bundles/org.openhab.binding.luxtronikheatpump/README.md:255]: Perhaps the read only value is currently only used by the UI to disable controls for changing the value.
Do you think it is a good idea to change the channel type @sgiehl? Maybe some others also need to be fixed.

Comment (sgiehl) [reply bundles/org.openhab.binding.luxtronikheatpump/README.md:255]: Seems I made a mistake with those channels while renaming them in the inital PR.
The heatpump actually serves a readonly and a writable channel that both serve the same value. I thought I had removed the readonly one so only the writable is available, as having both doesn't make much sense. It's correct that the channel `temperatureTargetHotWater_2` should be removed/replaced with `temperatureHotWaterTarget`.
@martin1 could you also remove those lines: https://github.com/openhab/openhab-addons/blob/cfea5a2185929a5a2172d7c5159232b5a2b5c5ae/bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml#L63-L69
And rename the channel (id & label) here: https://github.com/openhab/openhab-addons/blob/cfea5a2185929a5a2172d7c5159232b5a2b5c5ae/bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml#L1964-L1969

**Review by sgiehl — APPROVED** [review]

**Comment by fwolter** [comment]
@sgiehl Can you approve the latest commit?

**Review by sgiehl — APPROVED** [review]
looks good to me

**Pull request merged**